frontend/.gitignore

```diff
@@ -0,0 +1,66 @@
+# Logs
+logs
```

Comment:
We didn't have a ,gitignore before :O?

Reply:
It was generated by the Firebase CLI, we can go back and remove unnecessary rules